src/components/ImageView/index.js

```diff
@@ -266,7 +267,8 @@ class ImageView extends PureComponent {
                 onLayout={this.onContainerLayoutChanged}
                 style={[styles.imageViewContainer, styles.overflowAuto, styles.pRelative]}
             >
-                <Pressable
+                <PressableWithoutFeedback
+                    accessible={false}
```

Comment:
Is there a reason we're setting this to false?
We could make this accessible with `accessibilityRole="image"` and put file name in `accessibilityLabel`

Reply:
Good point, I just added it